support/openapi-template/Cargo.mustache

```diff
@@ -90,3 +110,5 @@ mockall = ["dep:mockall"]
 bon = ["dep:bon"]
 {{/useBonBuilder}}
 {{/reqwestTrait}}
+tsify = { workspace = true, optional = true, features = ["js"], default-features = false }
+wasm-bindgen = { workspace = true, optional = true, features = ["serde-serialize"] }
```

Comment:
We'd need some way to configure this in runtime, since this would now also apply to identity.